Testing/xASL_test_SPM.m

```diff
@@ -0,0 +1,125 @@
+function xASL_test_SPM(TestDirDest, bTestDataUsed)
```

Comment:
can you please add a header to this function.

Reply:
I have added the header.